### ISSUE TITLE
Wrong number of args error when calling single-argument version of close!

### ISSUE THREAD

**Issue by paulbutcher** [sunng87/ring-jetty9-adapter#70]
Calling the single-argument version of `close!` gives:

```
#error {
 :cause Wrong number of args (1) passed to: ring.adapter.jetty9.websocket/eval8784/fn--8789
 :via
 [{:type clojure.lang.ArityException
   :message Wrong number of args (1) passed to: ring.adapter.jetty9.websocket/eval8784/fn--8789
   :at [clojure.lang.AFn throwArity AFn.java 429]}]
 :trace
 [[clojure.lang.AFn throwArity AFn.java 429]
  [clojure.lang.AFn invoke AFn.java 32]
  [ring.adapter.jetty9.websocket$eval8566$fn__8651$G__8549__8662 invoke websocket.clj 19]
```

**Comment by sunng87** [comment]
I may hit a clojure bug when copying protocol functions like `(def close! ws/close!)`, only one form was copied.

**Comment by sunng87** [comment]
The fix was released in 0.17.4

**Comment by paulbutcher** [comment]
Thanks @sunng87, just tried it out and it looks good 👍
